website/docs/concepts/recorder_service/README.md

```diff
@@ -159,10 +189,10 @@ For example:
 
 ## How to Use
 
-Data can be recorded through the Recorder App, or using the `EventsClient` class.
+Data can be recorded through the App App, or using the `EventsClient` class.
```

Comment:
```suggestion
Data can be recorded through the AppBar, the Camera App, or using an `EventClient` as in the **Events Recorder example**. 
```

Reply:
Thanks for catching this. Mentioned camera app here: 356ab79d0af823a4e2c1306ff7e34c5943377ba6